src/app/authorization/registration/registration.component.css

```diff
@@ -1,33 +1,16 @@
-*{
-    font-family: Roboto;
+* {
+  font-family: Roboto;
```

Comment:
Why giving all font family to roboto ?

Reply:
because want same font family to the whole project